chrome/content/zotero/itemTree.jsx

```diff
@@ -3828,6 +3788,11 @@ var ItemTree = class ItemTree extends LibraryTree {
 		}
 		return icon;
 	}
+
+	_canCalculateAttachmentState(item) {
```

Comment:
→ `_canGetBestAttachmentState()`? `calculate` kind of makes it sound like something else.